core/src/main/scala/cats/syntax/parallel.scala

```diff
@@ -100,6 +113,19 @@ final class ParallelTraversableOps[T[_], A](private val ta: T[A]) extends AnyVal
 
 }
 
+final class ParallelTraverseFilterOps[T[_], A](private val ta: T[A]) extends AnyVal {
```

Comment:
I think it'd be better to consolidate these, so that all enrichment methods for `T[A]` are on a single enrichment class, mostly in order to reduce the number of implicits in `cats.syntax.all`, but also just to simplify this code. In this case that would mean putting these methods on something named `ParallelTraversableOps` or `ParallelFoldMapAOps`, though, which also seems kind of bad.

I think for now this version is probably best, but we should definitely revisit this in Cats 3.